### DISCUSSION TITLE
V0.0.6

### DISCUSSION THREAD

**Discussion by MFn233** [MFn233/PCL-Mainpage#8] [公告]
0.0.6更新，他终于来了。。。
**主页预设无法加载问题已经修复！！！！**接下来的更新将会继续同步至cdn @LTCatt

以下是新增加的一些功能：
1.添加了**网址栏**以弥补网页捷径不够多的问题，同时避免网页捷径过多导致的臃肿。但注意，**请不要顺手按下enter键打开网页，因为那样会直接启动游戏。**
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/e7eb031c-cb59-429a-bd39-39c0ea6db44b)

2.根据@MCSteve123的建议，在“杂项”页增加了**关于中文mcwiki迁移的通知**
![image](https://github.com/MFn233/PCL-Mainpage/assets/103323756/cbd1ae75-edc4-499b-9c52-37f483cdc5c1)

3.关于作者一栏，在鸣谢处增加了@JingHai-Lingyun

这是一次小更新，因为此前**多半在解决无法访问的问题**。

同时，**我将会在不久之内为该主页编写介绍页面**，敬请期待。

<hr /><em>This discussion was created from the release <a href='https://github.com/MFn233/PCL-Mainpage/releases/tag/V0.0.6'>V0.0.6</a>.</em>

**Comment by SqrtTan** [comment]
![image](https://github.com/MFn233/PCL-Mainpage/assets/121204926/84852956-71ff-4b90-a7f7-c58890b48430)
Q：
请从此文件同步消息中推测作者的心情。
A：
叹号连用，强调了兴奋、激动的语气，突出了作者的欣喜，表达了作者对完成一次更新的欣慰，对继续从事咕咕咕事业的的坚定不移……~（好吧实在编不下去了）~

**Reply by MFn233** [reply]
XD
